src/actions/types.js

```diff
@@ -1,5 +1,5 @@
 export const FETCH_FORECAST_REQUEST = 'FETCH_FORECAST_REQUEST';
-export const FETCH_FORECAST_START = 'FETCH_FOREACST_START';
+export const FETCH_FORECAST_START = 'FETCH_FORECAST_START';
```

Comment:
Can you explain me why this is required?

Reply:
Sure, I tried to use just one const to handle everything but I have an issue with saga.

FETCH_FORECAST_REQUEST is used to trigger saga and all the API related operations.
I use FETCH_FORECAST_START to display spinner (or otherwise notify interface that API request has started).

When I used one const for both and an action creater I was creating an infinite loop with saga.

I managed to get rid of the FETCH_FORECAST_END though as this was indeed not required.